src/database/initialization/organization_to_json.sql

```diff
@@ -1,11 +1,40 @@
-CREATE OR REPLACE FUNCTION organization_to_json(organization organizations)
+-- An explicit DROP is needed to remove the old function instead of overloading it.
+DROP FUNCTION IF EXISTS organization_to_json(organizations);
```

Comment:
This little bit caught me off guard: if you change the function parameters, that's a new function despite the same name. So you can end up having multiple functions unintentionally if `replace` was intended to replace by name.